pkg/base/linuxutils/partitionhelper/partition_helper.go

```diff
@@ -65,9 +65,9 @@ const (
 	// fill device and partition table type
 	CreatePartitionTableCmdTmpl = sgdisk + "%s -o"
 	// CreatePartitionCmdTmpl create partition on provided device cmd template, fill device and partition label
-	CreatePartitionCmdTmpl = sgdisk + "-a1 -n 1:0:0 -c 1:%s %s"
+	CreatePartitionCmdTmpl = sgdisk + "-n 1:0:0 -c 1:%s %s"
 	// CreatePartitionCmdWithUUIDTmpl create partition on provided device with uuid cmd template, fill device and partition label
-	CreatePartitionCmdWithUUIDTmpl = sgdisk + "-a1 -n 1:0:0 -c 1:%s -u 1:%s %s"
```

Comment:
Why `-a1` flag was set first time?

Reply:
Because I optimized to use as much sectors as possible and at local environment it gives the same behavior as parted. Also it wasn't noticed at some types of disks while testing real env.

Reply:
Ok